packages/widgets/src/components/pie/pie-chart.widget.utils.ts

```diff
@@ -1,9 +1,11 @@
+import { colors as palette } from '@keen.io/colors';
+
 export const createLegendLabels = (
```

Comment:
Based on provided stacked slices - removed single data series and push to legend series collection OTHERS slice.